README.md

```diff
@@ -1,26 +1,43 @@
+# lazygit
 
-A simple terminal UI for git commands, written in Go with the [gocui](https://github.com/jroimartin/gocui 'gocui') library.
+![CI](https://github.com/jesseduffield/lazygit/workflows/Continuous%20Integration/badge.svg) [![Go Report Card](https://goreportcard.com/badge/github.com/jesseduffield/lazygit)](https://goreportcard.com/report/github.com/jesseduffield/lazygit) [![GolangCI](https://golangci.com/badges/github.com/jesseduffield/lazygit.svg)](https://golangci.com) [![GoDoc](https://godoc.org/github.com/jesseduffield/lazygit?status.svg)](http://godoc.org/github.com/jesseduffield/lazygit) [![GitHub tag](https://img.shields.io/github/tag/jesseduffield/lazygit.svg)]() [![TODOs](https://badgen.net/https/api.tickgit.com/badgen/github.com/jesseduffield/lazygit)](https://www.tickgit.com/browse?repo=github.com/jesseduffield/lazygit)
+
+A simple terminal UI for git commands, written in Go with the [gocui](https://github.com/jroimartin/gocui "gocui") library.
 
 Rant time: You've heard it before, git is _powerful_, but what good is that power when everything is so damn hard to do? Interactive rebasing requires you to edit a goddamn TODO file in your editor? *Are you kidding me?* To stage part of a file you need to use a command line program to step through each hunk and if a hunk can't be split down any further but contains code you don't want to stage, you have to edit an arcane patch file _by hand_? *Are you KIDDING me?!* Sometimes you get asked to stash your changes when switching branches only to realise that after you switch and unstash that there weren't even any conflicts and it would have been fine to just checkout the branch directly? *YOU HAVE GOT TO BE KIDDING ME!*
 
 If you're a mere mortal like me and you're tired of hearing how powerful git is when in your daily life it's a powerful pain in your ass, lazygit might be for you.
 
 ![Gif](/docs/resources/lazygit-example.gif)
 
-- [Installation](https://github.com/jesseduffield/lazygit#installation)
-- [Usage](https://github.com/jesseduffield/lazygit#usage),
-  [Keybindings](/docs/keybindings)
-- [Cool Features](https://github.com/jesseduffield/lazygit#cool-features)
-- [Contributing](https://github.com/jesseduffield/lazygit#contributing)
-- [Video Tutorial](https://youtu.be/VDXvbHZYeKY)
-- [Rebase Magic Video Tutorial](https://youtu.be/4XaToVut_hs)
-- [Twitch Stream](https://www.twitch.tv/jesseduffield)
-
-[<img src="https://i.imgur.com/sVEktDn.png">](https://youtu.be/CPLdltN7wgE)
+## Table of contents
+
+- [Installation](#installation)
+  - [Homebrew](#homebrew)
+  - [MacPorts](#macports)
+  - [Ubuntu](#ubuntu)
+  - [Void Linux](#void-linux)
+  - [Scoop (Windows)](#scoop-(windows))
+  - [Arch Linux](#arch-linux)
+  - [Fedora and CentOS 7](#fedora-and-centos-7)
+  - [Conda](#conda)
+  - [Go](#go)
+  - [Binary releases](#binary-releases)
+- [Usage](#usage)
+  - [Keybindings](#keybindings)
```

Comment:
if you scroll down you'll see I've recently added an undo/redo usage link, so I would chuck that up here as well